README.md

```diff
@@ -136,6 +136,27 @@ required plugins.
 
 ### Intellij
 
+#### Configure Project JDK to Java 8 (1.8) JDK
+
+1. Open Project Settings dialog by going to  
+   `File -> Project Structure -> Project Settings -> Project`.
+   
+2. Click on the JDK version drop down and select `Download JDK...` or choose an existing recent Java 8 (1.8) JDK version.
+
+3. In the download dialog, select version `1.8` and vendor `AdoptOpenJDK (Hotspot)` for the JDK to download.
```

Comment:
I am not sure we should suggest a particular distribution.
I like AdoptOpenJDK but it is not required

Reply:
I have rephrased the wording to address your comment. PTAL